tests/test_models.py

```diff
@@ -1,40 +1,153 @@
+        self.assertEqual(customer.country, "Colombia")
+        self.assertEqual(customer.zipcode, "11023")
+
+    def test_add_a_customer(self):
+        """It should Create a customer and add it to the database"""
+        customers = Customer.all()
+        self.assertEqual(customers, [])
+        customer = Customer(name="Katerine",lastname="Perdomo",email="kate@email.com",phone="XXXXXX",address="casa",city="Bogota",state='CU',country="Colombia",zipcode="11023")
+        self.assertTrue(customer is not None)
+        self.assertEqual(customer.id, None)
+        customer.create()
+        # Assert that it was assigned an id and shows up in the database
+        self.assertIsNotNone(customer.id)
+        customers = Customer.all()
+        self.assertEqual(len(customers), 1)
+    
+    def test_read_a_customer(self):
+        """It should Read a Customer"""
+        customer = CustomerFactory()
+        logging.debug(customer)
+        customer.id = None
+        customer.create()
+        self.assertIsNotNone(customer.id)
+        # Fetch it back
+        found_customer = Customer.find(customer.id)
+        self.assertEqual(found_customer.id, found_customer.id)
+        self.assertEqual(found_customer.name, found_customer.name)
+        self.assertEqual(found_customer.lastname, customer.lastname)
+
+    def test_update_a_customer(self):
+        """It should Update a Customer"""
+        customer = CustomerFactory()
+        logging.debug(customer)
+        customer.id = None
+        customer.create()
+        logging.debug(customer)
+        self.assertIsNotNone(customer.id)
+        # Change it an save it
+        customer.lastname = "Moreno"
+        original_id = customer.id
+        customer.update()
+        self.assertEqual(customer.id, original_id)
+        self.assertEqual(customer.lastname, "Moreno")
+        # Fetch it back and make sure the id hasn't changed
+        # but the data did change
+        customers = Customer.all()
+        self.assertEqual(len(customers), 1)
+        self.assertEqual(customers[0].id, original_id)
+        self.assertEqual(customers[0].lastname, "Moreno")
+
+    def test_update_no_id(self):
+        """It should not Update a Customer with no id"""
+        customer = CustomerFactory()
+        logging.debug(customer)
+        customer.id = None
+        self.assertRaises(DataValidationError, customer.update)
+    
+    def test_delete_a_customer(self):
+        """It should Delete a Customer"""
+        customer = CustomerFactory()
+        customer.create()
+        self.assertEqual(len(Customer.all()), 1)
+        # delete the pet and make sure it isn't in the database
+        customer.delete()
+        self.assertEqual(len(Customer.all()), 0)
+    
+    def test_list_all_customers(self):
+        """It should List all Customers in the database"""
+        customers = Customer.all()
+        self.assertEqual(customers, [])
+        # Create 5 Customers
+        for _ in range(5):
```

Comment:
Probably better to use a variable here for the number of new customers and the expected number to be returned. 
ie, num_new_customers = 5